packages/design-system/src/components/Banner/Banner.md

```diff
@@ -0,0 +1,136 @@
+### Banner - Notifications
```

Comment:
I think it should be just `Banner` and it should be placed under `Alerts` category in docs.

Reply:
I agree, it the same type of component as Toasts and gives feedback to the user.

Reply:
done